app/Services/ServerService.php

```diff
@@ -137,15 +137,24 @@ public function getSourceQueryInformation(Server $server): array
                 $this->getServerIp($server),
                 $this->getPort($server),
                 1,
-                $this->sourceQuery::GOLDSOURCE
+                $this->sourceQuery::SOURCE
```

Comment:
I think we need to add a new field to the servers table so that you can choose different engine for each server.

Reply:
I think current GoldSrc servers use the same Source response protocol: https://developer.valvesoftware.com/wiki/Server_queries#Obsolete_GoldSource_Response Although it would not hurt to add it, someone interested may run an old server.